cleaners/firefox.xml

```diff
@@ -18,51 +18,133 @@
     You should have received a copy of the GNU General Public License
     along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
+    @app Mozilla Firefox
+    @url https://www.mozilla.org/en-US/firefox/new/
+    @os Windows, Linux, FreeBSD, OpenBSD
+    @cleanerversion v2.1.1
+    @cleanerdate 2019-03-29
+    @cleanerby Andrew Ziem (2018-10-01 - 2018-11-05) & luoliyan (2018-10-31) & danielk43 (2018-11-29) & Tobias B. Besemer (2019-02-12 - 2019-03-29)
+    @tested ok v67.0a1, Windows 7
+    @testeddate 2019-03-15 (... with an old version of the cleaner!)
+    @testedby https://github.com/Tobias-B-Besemer
+    @note The cleaner is for different editions, like the normal "Mozilla Firefox" edition and e.g. the "Mozilla Firefox Developer Edition"...
 
+-->
+<!--
+This cleaner includes first steps to make a universal "All Firefox based browsers"-Code-Base!
+I will fork all_firefox_based.xml later from this file.
+And then fork for each browser a file from all_firefox_based.xml.
+Tobias.
 -->
 <cleaner id="firefox">
   <label>Firefox</label>
   <description>Web browser</description>
   <running type="exe" os="unix">firefox</running>
   <running type="exe" os="unix">firefox-bin</running>
   <running type="exe" os="windows">firefox.exe</running>
+  <!--
+  Just a guess! I will figure this out later! Tobias.
+  <running type="exe" os="linux">firefox</running>
+  -->
+  <!-- If you already wanna give the universal cleaner a try: Remove the comment-marks from the following block... -->
+  <!-- The following values are just guesses... -->
+  <!--
+  <running type="exe" os="windows">seamonkey.exe</running>
+  <running type="exe" os="linux">seamonkey-bin</running>
+  -->
+  <!--
   <running type="pathname">~/.mozilla/firefox/*.default/lock</running>
+  
+  What is, if the user have multiple profiles that he maybe have renamed ???
+  Or from different versions of FF ??? (E.g. Developer Edition...)
+  On Windows I have profile folders that have e.g. the following names:
+  - zypzyju5.default
+  - zxzu78e2.dev-edition-default (From the Developer Edition !!!)
+  - uaj16i8x.Default User (Some day renamed ???)
+  - 5umya027.Test (Surely renamed...)
+  
+  So I guess the line should be:
+  <running type="pathname" search=glob">~/.mozilla/firefox/*/lock</running>
+  
+  But as we already check if the executable is running, I prefer to remove this line...
+  -->
+  <var name="Browser">
```

Comment:
To get this working, I removed this new multi-value variable, so there were no nested multi-value variables

Reply:
I see there no multi value variable... just a "*" in the path...